front/app/containers/IdeasNewPage/IdeasNewForm/Heading.tsx

```diff
@@ -55,6 +58,10 @@ export const Heading = ({
   };
   const isSurveyOnMobile = isSmallerThanPhone && isSurvey;
 
+  const goBackToProject = useCallback(() => {
```

Comment:
And here - the use callback does not seem necessary. In general, let's avoid memoizing unless there's a good reason to do it. It adds complexity to the code without adding much performance value in most cases.